components/layout_2020/display_list/mod.rs

```diff
@@ -1300,3 +1297,84 @@ fn resolve_border_image_slice(
         resolve_percentage(border_image_slice.3, size.width),
     )
 }
+
+pub(crate) fn normalize_radii(rect: &units::LayoutRect, radius: &mut wr::BorderRadius) {
```

Comment:
Is it possible to limit the scope of the visibility here? I think this is only useful for display list building.

```suggestion
pub(super) fn normalize_radii(rect: &units::LayoutRect, radius: &mut wr::BorderRadius) {
```